modules/common/src/main/java/com/opengamma/sdk/common/ServiceInvoker.java

```diff
@@ -43,6 +43,10 @@
  * such as via the try-with-resources statement.
  */
 public final class ServiceInvoker implements AutoCloseable {
+  /**
+   * The User Agent header to attach to all outgoing HTTP requests.
+   */
+  public static final String USER_AGENT = "OG-JAVA-SDK";
```

Comment:
The approach I've seen most (and which works most reliably) is to create a `version.properties` file and update the data using maven. See method #3 [here](http://blog.soebes.de/blog/2014/01/02/version-information-into-your-appas-with-maven/). Note that we can use `PropertyFile` to load the file. When not in a jar file, the property may not be set, so the code should handle that.